src/rich-text-editor.component.js

```diff
@@ -28,7 +28,7 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
     // https://developer.mozilla.org/en-US/docs/Web/Events/paste
     event.preventDefault()
     event.stopPropagation()
-    let paste = richTextContext.sanitizeHTML((window.clipboardData || event.clipboardData).getData('text/html'), 'pasteHTML')
+    let paste = richTextContext.sanitizeHTML((window.clipboardData || event.clipboardData).getData('text/plain'), 'pasteHTML')
```

Comment:
We should parse the `getData` format as `text/plain` instead of `text/html` to ensure it has a chance to go through the sanitizeHTML function first before being deserialized as html.